feat(core): add an option for preventing window overflow (#9687)
tauri-apps/tauri: .changes/prevent-window-overflow.md

---
"tauri": "patch:feat"
"tauri-utils": "patch:feat"
"tauri-runtime": "patch:feat"
"tauri-runtime-wry": "patch:feat"
"@tauri-apps/api": "patch:feat"
---

> Add an option `prevent_overflow` for preventing the window frin overflowing the monitor on creation

Reviewer: ```suggestion
Add an option `prevent_overflow` for preventing the window from overflowing the monitor on creation
```

> Add an option `prevent_overflow` for preventing the window from overflowing the monitor on creation

Reviewer 2: ```suggestion
Add `preventOverflow` config option to prevent the window from overflowing the monitor size on creation
```
